src/main/java/luke/exception/LukeException.java

```diff
@@ -0,0 +1,7 @@
+package luke.exception;
+
+public class LukeException extends RuntimeException {
```

Comment:
Not too sure but IIRC it's safer to inherit from `Exception` rather than `RuntimeException`?

In case you forget to catch a LukeException the compiler can warn you then.

Reply:
Oh okay. I will try replacing the RuntimeException with Exception!